matrix_client/room.py

```diff
@@ -226,6 +244,20 @@ def send_audio(self, url, name, **audioinfo):
         return self.client.api.send_content(self.room_id, url, name, "m.audio",
                                             extra_information=audioinfo)
 
+    def send_encrypted(self, content):
+        """Send an arbitrary encrypted message to the room.
+
+        Args:
+            content (dict): The content of a m.room.message event.
+
+        Raises:
+            RuntimeError if encryption is not enabled in client, or if the room
+            is unencrypted.
+        """
+        if not self.client._encryption or not self.encrypted:
+            raise RuntimeError
```

Comment:
Is `RuntimeError` the best exception to be raised here (it might be, but I've never used it before, so I don't know when it'd be appropriate)? My approach would have been to define a new `MatrixNotEncrypted` exception or something along those lines.

Reply:
You're right, I've been lazy (and wasn't sure if it was worth a new exception). Note that the doc says about [`RuntimeError`](https://docs.python.org/3/library/exceptions.html#RuntimeError) "Raised when an error is detected that doesn’t fall in any of the other categories.", so it was probably the best built-in exception :)